packages/terra-slide-panel/src/SlidePanel.jsx

```diff
@@ -114,5 +138,6 @@ class SlidePanel extends React.Component {
 
 SlidePanel.propTypes = propTypes;
 SlidePanel.defaultProps = defaultProps;
+SlidePanel.position = position;
```

Comment:
Why have you assigned it to the Class here?

Reply:
I have assigned it to the class for the purpose of exposing the enum.
Since the "position" enum will only be referenced for SlidePanel, assigning it to the class prevents us from explicitly importing the enum.
Reference : [https://jaketrent.com/post/expose-enum-props-in-react/](url)

Reply:
Let's update to export slidepanel positions like so:

```jsx
export default SlidePanel;
export { SlidePanelPositions }
```
This aligns to the pattern we talked about here: https://github.com/cerner/terra-core/issues/1044#issuecomment-386711960

Reply:
Updated to export the enum separately as suggested by @bjankord.